erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -285,7 +285,7 @@ def get_invoice_details(self):
 
 	def set_party_details(self, row):
 		# customer / supplier name
-		party_details = self.get_party_details(row.party)
+		party_details = self.get_party_details(row.party) or {}
```

Comment:
For some reason (probably due to customer's data) `get_party_details` was returning `None`